plugins/ocm-backend/dist-dynamic/package.json

```
@@ -62,17 +62,15 @@
   "bugs": "https://github.com/janus-idp/backstage-plugins/issues",
   "bundleDependencies": true,
   "peerDependencies": {
-    "@backstage/backend-common": "^0.19.8",
-    "@backstage/backend-plugin-api": "^0.6.6",
-    "@backstage/backend-tasks": "^0.5.11",
-    "@backstage/catalog-client": "^1.4.5",
+    "@backstage/backend-common": "^0.20.1",
+    "@backstage/backend-plugin-api": "^0.6.9",
+    "@backstage/backend-tasks": "^0.5.14",
+    "@backstage/catalog-client": "^1.5.2",
     "@backstage/catalog-model": "^1.4.3",
     "@backstage/config": "^1.1.1",
-    "@backstage/plugin-catalog-node": "^1.4.7",
+    "@backstage/plugin-catalog-node": "^1.6.1",
     "@backstage/backend-plugin-manager": "npm:@janus-idp/backend-plugin-manager@0.0.2-janus.5",
```

```suggestion
    "@backstage/backend-dynamic-feature-service": "0.1.0"
```

Of course the import in the code should be changed as well.
This package is now part of the upstream backstage since Backstage 1.22.0.